HIP/hip-646.md

```diff
@@ -94,6 +94,102 @@ message TokenInfo {
 Pricing considerations:
 The price for this metadata will be identical to the cost for storage of the memo.
 
+### Hedera REST API Changes
+
+Only for individual NFTs does the Mirror Node REST API currently return a base64 encoded representation of the `metadata` field. This will be extended to an entire token type, both fungible (this HIP) and non-fungible token collections ([HIP-765](https://hips.hedera.com/hip/hip-765)).
+
+#### /api/v1/tokens
+
+```json
+{
+    "links": {
+        "next": "/api/v1/tokens?token.id=gt:0.0.1107"
```

Comment:
It's not clear if the next link is accurate without every item, so better to just null it.
```suggestion
        "next": null
```

Reply:
Done